test-requirements.txt

```diff
@@ -13,7 +13,6 @@ pytest>=6.2.4
 pytest-xdist>=1.34.0
 pytest-forked>=1.3.0,<2.0.0
```

Comment:
Didn't check but you might not be using this one as well; it used to be a dep of pytest-xdist but isn't anymore, and is not used much by itself.

Reply:
Oh, thanks -- let me check that

Reply:
Indeed, looks like we don't actually use it, and only ever added it to `test-requirements.txt` in order to pin a specific version to fix an issue where tests were failing: https://github.com/python/mypy/commit/e006b94e097184451e6baf9b9c55e56176728ab8.

Thanks!